docs/migrations/v6-to-v7.md

```diff
@@ -13,7 +13,31 @@ There are four sections based on the four potential user groups of this document
 
 ## Chains
```

Comment:
the headline above needs to be updated (currently it's `Migrating from ibc-go v5 to v6`)